src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/naming/catchparametername/Example1.java

```diff
@@ -6,32 +6,37 @@
 </module>
 */
 
+package com.puppycrawl.tools.checkstyle.checks.naming.catchparametername;
+
 // xdoc section -- start
-public class MyTest {
+public class Example1 {
   public void myTest() {
     try {
       // ...
+      throw new InterruptedException();
     } catch (ArithmeticException e) { // OK
       // ...
     } catch (ArrayIndexOutOfBoundsException ex) { // OK
       // ...
-    } catch (Throwable t) { // OK
-      // ...
-    } catch (IndexOutOfBoundsException e123) { // violation, digits
-                                               // not allowed
+    } catch (IndexOutOfBoundsException e123) {
+      // violation above, 'Name 'e123' must match pattern'
+      // not allowed
       // ...
-    } catch (NullPointerException ab) { // violation, should have at least
-                                        // three characters if not e|t|ex
+    } catch (NullPointerException ab) {
+      // violation above, 'Name 'ab' must match pattern'
       // ...
     } catch (ArrayStoreException abc) { // OK
       // ...
-    } catch (InterruptedException aBC) { // violation, first two characters
-                                         // should be in lowercase
+    } catch (InterruptedException aBC) {
+      // violation above, 'Name 'aBC' must match pattern'
+      // should be in lowercase
       // ...
     } catch (RuntimeException abC) { // OK
       // ...
     } catch (Exception abCD) { // OK
       // ...
+    } catch (Throwable t) { // OK
```

Comment:
```suggestion
    } catch (Throwable t) {
```

Please remove all `// OK` comments in all examples.

Reply:
Removed all //OK comment and pushed the changes